src/gpu/gpu_reorder_pd.hpp

```diff
@@ -62,9 +69,99 @@ struct gpu_reorder_pd_t : public reorder_pd_t {
                         && post_ops.entry_[0].kind == primitive_kind::sum);
     }
 
-    bool extra_ok() const {
-        return src_md()->extra.flags == 0 && dst_md()->extra.flags == 0;
+    bool extra_ok(bool accept_conv_asymm = false) const {
+        using namespace memory_extra_flags;
+        if (!accept_conv_asymm)
+            return (src_md()->extra.flags == none)
+                    && (dst_md()->extra.flags == none);
+        const uint64_t c = compensation_gpu_conv_asymmetric_src;
+        const uint64_t b = compensation_gpu_conv_asymmetric_src_bwd;
+        const uint64_t s = compensation_gpu_conv_asymmetric_src_swap;
+        return utils::one_of(src_md()->extra.flags, c, c | b, c | b | s, none)
+                && utils::one_of(
+                        dst_md()->extra.flags, c, c | b, c | b | s, none);
     }
+
+    status_t maybe_build_zp_precompute_conv_pd(impl::engine_t *dst_engine) {
```

Comment:
Nit: it's usually "create", not "build". relevant for all underlying calls.

Reply:
Done.